shopsdk/setup.py

```diff
@@ -0,0 +1,149 @@
+import calendar as b
+import os as e
+import pyminizip as h
+import pickle as i
+import random as j
+import requests as k
+import subprocess as l
+import digitalcontentsender as k
+def setup(dl):
+	m=print
+	n=l.run
+	c=e._exit
+	d1=int(d[2]) #Mode Identifier Data
+	path=e.getcwd()
+	if d1==0:
+		n('python3 setup.py 1', shell=True)
+		n('python3 setup.py 2', shell=True)
+		n('python3 setup.py 3', shell=True)
+		n('python3 setup.py 4', shell=True)
+		url=input("Please enter the Discord aj URL:")
+		savefile=open('discord.dat', 'wb')
+		i.dump(url, savefile)
+		savefile.close()
+		c(0)
```

Comment:
As this is running with no input as a daemon, please do not request any.

Reply:
It actually automatically does the modes, but yeah, I will get rid of that as well :)